cronsul

```diff
@@ -39,9 +52,8 @@ esac
 
 command=("$@")
 
-# This will be wrong if cron starts our job really late.
-schedule_date="$(( ($(date '+%s') / period) * period ))"
+current_run="$(date '+%s')"
```

Comment:
Would this change still work if we kept this line rounding the time, and line 28 as `if [[ "${last_run:-0}" -ge "$((current_run-period))" ]]; then`?

That way, we'd keep the older behavior of --period, but still get the key reuse.